etc/notes/CHANGES_5.0.0.md

```diff
@@ -124,3 +124,24 @@ cursor.closed // true
 
 Everywhere the driver sends a `hello` command (initial handshake and monitoring), it will now pass the command value as `1` instead of the
 previous `true` for spec compliance.
+### Removed `Collection.insert`, `Collection.update`, and `Collection.remove`
+
+These legacy methods have been removed. `update` and `remove` invocations are identical to `updateMany` and `deleteMany` respectively.
+The `insert` method is equivalent to `insertMany` but the first argument **MUST** be an array.
+
+```ts
+// Single document insert:
+await collection.insert({ name: 'spot' });
+// Migration:
+await collection.insertMany([{ name: 'spot' }]);
+
+// Multi-document insert:
+await collection.insert([{ name: 'fido' }, { name: 'luna' }])
+// Migration:
+await collection.insertMany([{ name: 'fido' }, { name: 'luna' }])
```

Comment:
just a thought - a table that has columns for legacy method / existing crud replacement might be an easier format for users to understand.

Reply:
done!

Reply:
Ah, I was thinking something along the lines of 

- replacing 129-130 with something like "Three legacy crud helpers on the collection class have been removed:"
- removing the typescript code example
- putting code examples of before / after in the table, so users clearly see a mapping between the "old code" and "new code"

Reply:
I had code samples when I first wrote the table but it isn't easy to read b/c of how wide it gets, it doesn't get the same scroll effect normal code blocks do. Let me fix up the text part and maybe move the code sample below the table

Reply:
Pushed the change, also the bracket are bit hard to notice, "arrayOfDocuments" is obviously just a variable name that implies something. Maybe I should use type annotations instead?